h/static/styles/app.scss

```diff
@@ -117,6 +118,9 @@ $group-list-width: 270px;
 
   .dropdown-menu {
     width: $group-list-width;
+    max-height: 500px;  // fallback for browsers lacking support for vh/calc
+    max-height: calc(100vh - #{$top-bar-height} - #{$group-list-spacing-below});
+    overflow-y: scroll;
```

Comment:
I think only showing scrollbars when the group list needs them, using `overflow-y: auto` looks better in desktop browsers. On mobile you'll get the same behavior either way.